docs/book/src/component-config-tutorial/testdata/project/README.md

```diff
@@ -75,7 +75,7 @@ More information can be found via the [Kubebuilder Documentation](https://book.k
 
 ## License
 
-Copyright 2023 The Kubernetes authors.
+Copyright 2024 The Kubernetes authors.
```

Comment:
@jonas-jonas 

Really thank you for that.
Could you please push a PR only with this change?
Then, I can approve it asap. 

We should have one 1 solution/issue per PR 
and only one commit either.

Reply:
Already done, here: https://github.com/kubernetes-sigs/kubebuilder/pull/3722

Reply:
Ops I saw: https://github.com/kubernetes-sigs/kubebuilder/pull/3722

approved/lgtm that one